MATLAB.gitlab-ci.yml

```diff
@@ -110,3 +111,23 @@ test_artifacts:
 #
 # [8] https://www.mathworks.com/help/matlab/ref/matlab.unittest.plugins-package.html
 # [9] https://www.mathworks.com/help/matlab/matlab_prog/generate-artifacts-using-matlab-unit-test-plugins.html
+
+# Starting in R2022b, the `build` job runs a build using the MATLAB build tool. You can use this job to run the 
+# tasks specified in a file named "buildfile.m" in the root of your repository. To use the job in your pipeline, 
+# substitute “mytask” with the tasks you want to run.
+#
+build:
+  extends: .matlab_defaults
+  script: matlab -batch "buildtool mytask"
+
+# If you specify more than one task, use a space to separate them. If you do not specify any tasks, the job 
+# runs the default tasks in "buildfile.m" as well as all the tasks on which they depend. The job calls the 
+# [`buildtool`][10] command to run a build using the tasks and options that you specify.
+#
+# MATLAB exits with exit code 0 if the build runs successfully. Otherwise, MATLAB terminates with a nonzero 
+# exit code, which causes the job to fail. When you use the `build` job, a file named "buildfile.m" must be in 
+# the root of your repository. For more information about the MATLAB build tool, 
+# see [`Create and Run Tasks Using Build Tool`][11].
```

Comment:
This text does not need to be enclosed with `` because it's not a literal.

Reply:
I'm confused about the backticks. They don't actually do anthing in this file, so I'm just trying to follow the same formatting that's used elsewhere. My choice would probably be to remove all the backticks if they're not doing anything but I'm not sure what purpose they actually serve

Reply:
I think we need to keep the backticks for any literal term because GitLab itself uses them. For an example, see the first two lines of this template to see how GitLab refers to filenames and keywords in its own boilerplate text: https://gitlab.com/gitlab-org/gitlab/-/blob/master/lib/gitlab/ci/templates/Elixir.gitlab-ci.yml

So please keep the backticks and refer to files (as shown above): `buildfile.m`

Reply:
Ah interesting, you're totally right. I wonder why they did that.